### ISSUE TITLE
Please Help ,Not Able to find exact reason so that cocoapod library not generating doc and directly navigate to github on click .

### ISSUE THREAD

**Issue by anutheriddle** [CocoaPods/cocoadocs.org#585]
I using xcode 9.2 to release FireStoreHelper cocoapod which is under progress .
i can see my library in search but when i click on that it direclty navigate to GitHub instead of description page and i am also not able ot see documentation .

I check by url provided but getting timeout error .

please help on this

**Comment by jalingo** [comment]
Having the same issue. Posted a [question on SO](https://stackoverflow.com/questions/48956794/cant-get-cocoadocs-page-for-swift-framework), but no one has responded yet.

If anyone can point me in the right direction. I believe I've exhausted the trouble shooting steps available in documentation.

Thanks in advance.

**Comment by sisoje** [comment]
its broken for weeks, nobody responds to tickets.
i suspect here is the key problem: https://cocoadocs.buddybuild.com/error/PromiseKit
happens for every pod

**Comment by paulb777** [comment]
From that log, looks like the CocoaPods gem needs to be updated to 1.4.0 to recognize `swift_version`.

**Comment by sisoje** [comment]
no, it has nothing to do with pods. i guess it has to do about this http://blog.cocoapods.org/CocoaDocs-Documentation-Sunsetting/

**Comment by paulb777** [comment]
From the PromiseKit log you posted above:` Invalid PromiseKit.podspec file: undefined method swift_version=` and `ruby/2.3.1/gems/cocoapods-1.3.1`
